libpod/pod_api.go

```diff
@@ -597,6 +599,15 @@ func (p *Pod) Inspect() (*define.InspectPodData, error) {
 		infraConfig.CPUSetCPUs = p.ResourceLim().CPU.Cpus
 		infraConfig.PidNS = p.PidMode()
 		infraConfig.UserNS = p.UserNSMode()
+		var nodes map[string]string
+		nodes, err = util.FindDeviceNodes()
+		if err != nil {
+			return nil, err
+		}
+		devices, err = infra.GetDevices(define.InspectContainerHostConfig{}, *infra.Spec(), nodes)
```

Comment:
```suggestion
		devices, err = infra.GetDevices(define.InspectContainerHostConfig{}, *infra.config.spec, nodes)
```
Please use `infra.config.spec`, the public accessors are super expensive because they use JSONDeepCopy